.codeclimate.yml

```diff
@@ -24,25 +21,24 @@ engines:
       - javascript
   eslint:
     enabled: false
-    channel: "eslint-3"
+    channel: eslint-3
   fixme:
-    # let's enable later
     enabled: false
   markdownlint:
-    # let's enable later
     enabled: false
-  rubocop:
-    enabled: true
-    config: '.rubocop_cc.yml'
-    channel: rubocop-0-69
 prepare:
   fetch:
-  - url: "https://raw.githubusercontent.com/ManageIQ/guides/master/.rubocop_base.yml"
+  - url: https://raw.githubusercontent.com/ManageIQ/manageiq-style/master/.rubocop_base.yml
     path: ".rubocop_base.yml"
-  - url: "https://raw.githubusercontent.com/ManageIQ/guides/master/.rubocop_cc_base.yml"
+  - url: https://raw.githubusercontent.com/ManageIQ/manageiq-style/master/.rubocop_cc_base.yml
     path: ".rubocop_cc_base.yml"
 ratings:
   paths:
   - Gemfile.lock
   - "**.rake"
   - "**.rb"
+plugins:
+  rubocop:
+    enabled: true
+    config: ".rubocop_cc.yml"
+    channel: rubocop-0-82
```

Comment:
Can you upgrade to rubocop-1-0?

Reply:
We recently upgraded to 0.82.  We're working out some issues with codeclimate first, then we can investigate upgrading to 1.0.